Harvest4usMOBILE/app/src/main/java/com/example/myfirstapp/CartActivity.kt

```diff
@@ -196,27 +196,28 @@ class CartActivity : AppCompatActivity(), CellClickListener {
         }
     }
 
-    override fun onCellClickListener(data : ItemsViewModel) {
+    override fun onCellClickListener(data: ItemsViewModel) {
         Toast.makeText(this, data.produceType, Toast.LENGTH_SHORT).show()
     }
 
-    fun run(url: String) {
-        val request = Request.Builder()
-            .url(url)
-            .build()
-
-        client.newCall(request).enqueue(object : Callback {
-            override fun onFailure(call: Call, e: IOException) {}
-            override fun onResponse(call: Call, response: Response) {
-                responseString = response.body?.string()!!
-                println(responseString)
-            }
-        })
-    }
+    // DEPRECATED: Previous team's backend call
+//    fun run(url: String) {
```

Comment:
Would it be better to just delete the run function from all the files? Looks good otherwise

Reply:
I was leaving it in case we wanted to see how they were generating their response strings. But I mean, we could always just look at the git history so I can remove it to make it cleaner

Reply:
Either way works, I'm down for whatever you choose.